bravado/mapping/param.py

```diff
@@ -128,6 +132,73 @@ def marshal_param(param, value, request):
             format(location))
```

Comment:
The new version's L107 marshals the `Pet` model object into a dict.  By the time it gets to L128, it is a dict.  However, the parameter to `json.dumps` need not always be a dict.  int, bool, string, etc are totally legit json objects.